hack/lsplatform/lsplatform.go

```diff
@@ -12,17 +12,21 @@ import (
 
 func main() {
 	platformName := ""
+	platformVersion := ""
 	flag.StringVar(&platformName, "is-platform", "", "check if the detected platform matches the given one")
+	flag.StringVar(&platformVersion, "is-platform-version", "", "check if the detected platform version matches the given one")
```

Comment:
version checking is not yet needed for this tool, we can elide this change from this PR

Reply:
Okay, will remove.